### PR TITLE
T14639 packaging

### PR DESCRIPTION
**Pull request by xaiki**
we are upstream,
 - no need to break other PRs by silly name change, dh_python does the right thing...
 - no need to split debian packaging

### CODE REVIEW

File: dbus-service.py [hunk 1]
```diff
@@ -30,8 +30,8 @@
 
 from pyndn import Name, Face
 
-from eosdatadistribution.ndn import Consumer
```
Comment (magcius) [inline dbus-service.py]: You forgot to rename the files themselves.

File: scripts/edd-get-appdata [hunk 2]
```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
```
Comment (magcius) [inline scripts/edd-get-appdata]: Python convention is to have all code in packages / modules so we can test it, and then use entry_points. Not a fan of this commit.

File: dbus-service.py (hunk 1 above)
Comment (xaiki) [inline dbus-service.py]: i don't get that comment ?

Comment (magcius) [inline dbus-service.py]: You changed the imports back to eos_data_distribution but the files are still named eosdatadistribution in this commit.

See the tree: https://github.com/endlessm/endless-ndn/tree/49266c8b01291b7feb74d051160d848554f454b7

File: scripts/edd-get-appdata (hunk 2 above)
Comment (xaiki) [inline scripts/edd-get-appdata]: i followed this: https://docs.python.org/2/distutils/setupscript.html#installing-scripts
couldn't really find a better doc, happy to read more.

Comment (magcius) [inline scripts/edd-get-appdata]: See http://python-packaging.readthedocs.io/en/latest/command-line-scripts.html

File: dbus-service.py (hunk 1 above)
Comment (xaiki) [inline dbus-service.py]: squashed both commits.

**Comment by magcius** [comment]
Pushed entrypoints for the remaining scripts, except for get_subscription, since that's waiting for fixups in another PR.

**Pull request closed without merging**